### ISSUE TITLE
Metagame - Crash when return to the overworld after a Game Over

### ISSUE THREAD

**Issue by Armitage1982** [cortex-command-community/Cortex-Command-Community-Project-Data#110]
**Describe the bug**
Often, Metagame mode does not send any opponents. So this time I decided to send a vessel to retrieve my brain robot. After a bad maneuver, I blew up my unit which triggered a game over. Back on the main map the game simply crashed without any error message.

**To Reproduce**
1. Launch a Metagame
2. Scan a map
3. Hope it's a human vs A.I. combat
4. Observed that no enemies are sent
5. Crush Brain Robot with an empty rescue vessel
6. Game return to previous world map and crash

**Expected behavior**
Getting back to the overworld

**Log context**
ERROR: Base.rte/AI/HumanBehaviors.lua:2139: attempt to index local 'PrjDat' (a nil value)
PRINT: GlobalFlameHandler:Create()
ERROR: Base.rte/Activities/Survival.lua:89: attempt to call method 'IsPaused' (a nil value)
SYSTEM: Activity "Survival" was ended
ERROR: Base.rte/Activities/Survival.lua:89: attempt to call method 'IsPaused' (a nil value)
SYSTEM: Activity "Survival" was ended
PRINT: GlobalFlameHandler:Destroy()

**Comment by fourZK** [comment]
The console log describes the Survival activity, not metagame. Sadly, a hard crash doesn't update `LogConsole.txt`, but at least you managed to catch some other error in there :p

Anyways... metagame has unfortunately been overlooked for a while now, but it's good that you're reporting this bug on a development branch, I assume? Please do specify the version/branch as well, it helps!

**Comment by Armitage1982** [comment]
> The console log describes the Survival activity, not metagame. Sadly, a hard crash doesn't update `LogConsole.txt`, but at least you managed to catch some other error in there :p
> 
> Anyways... metagame has unfortunately been overlooked for a while now, but it's good that you're reporting this bug on a development branch, I assume? Please do specify the version/branch as well, it helps!

Yes, of course, I actually forgot to mention that it was on the last **4zk-content** branch ( https://github.com/cortex-command-community/Cortex-Command-Community-Project-Data/commit/316755990dc7f584238f07fc7a22f868bbc07e0d ) compiled from the **4zk-content-source** branch as of August 17th ( https://github.com/cortex-command-community/Cortex-Command-Community-Project-Source/commit/301c24185269123287728932edab91acfd61a8ff ).

Which by the way compile flawlessly with great work on the new interfaces :-)

I haven't seen any more information on the proper way to do a crash report. If there is one, I'd be happy to participate.
It's nice to see the game come back.
Thanks

**Comment by garethyr** [comment]
Thanks for the info on this. If this is happening in a dev branch then it's great to know about! And as long as this step-by-step can reliably reproduce the problem then that's all you need to do.

@MaximDude or @fourZK this is something that should be fixed for next version, regardless of code freeze, so people can stop having issues with the metagame. If one of you could take a look, that'll be great, otherwise I'll do it next weekend when I'm back in town.
